test/nightly/deploy.test.ts

```diff
@@ -19,36 +19,35 @@ interface ITestCase {
      */
     workspaceFolder: string | undefined;
     runtimePrefix: string;
-    runtimeSuffix?: string;
     versions: IVersionInfo[];
 }
 
 interface IVersionInfo {
     version: string;
     supportedOs: 'Windows' | 'Linux' | 'Both';
+    displayText?: string;
 }
 
 suite('Create Web App and deploy', async function (this: Mocha.Suite): Promise<void> {
-    this.timeout(12 * 60 * 1000);
```

Comment:
Because this timeout is applied per-test and it seemed ridiculous to let each test run for 12 minutes haha. Specifically, the .NET 5 tests were timing out (I think because .NET 5 support hasn't completely rolled out), and I had a build last 45 minutes: https://dev.azure.com/ms-azuretools/AzCode/_build/results?buildId=27863&view=results